drivers/hmis/app/models/hmis/hud/current_living_situation.rb

```diff
@@ -22,7 +22,7 @@ class Hmis::Hud::CurrentLivingSituation < Hmis::Hud::Base
   after_commit :warehouse_trigger_processing
 
   private def warehouse_trigger_processing
-    return unless warehouse_columns_changed?
+    return unless enrollment && warehouse_columns_changed?
```

Comment:
I believe this came up from the factory constructing the object.

I hadn't seen any errors like this come through sentry. This gets invoked after commit, and it wouldn't be possible to edit any of these records in the app because the app would fail to resolve them.